packages/transformer/src/test/standalone/IModelTransformerHub.test.ts

```diff
@@ -1662,6 +1662,122 @@ describe("IModelTransformerHub", () => {
+      { sourceLabel: "41", targetLabel: "42", idInBranch: "not inserted yet", sourceFedGuid: false, targetFedGuid: false },
+    ];
+
+    let aspectId: Id64String | undefined;
+    const timeline: Timeline = [
+      { master: { seed: masterSeed } },
+      { branch: { branch: "master" } },
+      {
+        branch: {
+          manualUpdate(db) {
+            expectedRelationships.map(
+              ({ sourceLabel, targetLabel }, i) => {
+                const sourceId = IModelTestUtils.queryByUserLabel(db, sourceLabel);
+                const targetId = IModelTestUtils.queryByUserLabel(db, targetLabel);
+                assert(sourceId && targetId);
+                const rel = ElementGroupsMembers.create(db, sourceId, targetId, 0);
+                expectedRelationships[i].idInBranch = rel.insert();
+              }
+            );
+          },
+        },
+      },
+      { master: { sync: ["branch"] } }, // first master<-branch reverse sync
+      {
+        assert({branch}) {
+          // expectedRelationships[1] has no fedguids, so expect to find an esa.
+          const sourceId = IModelTestUtils.queryByUserLabel(branch.db, expectedRelationships[1].sourceLabel);
+          aspectId = branch.db.elements.getAspects(sourceId, ExternalSourceAspect.classFullName)[0].id;
+          assert(Id64.isValid(aspectId));
+        },
+      },
+      {
+        master: {
+          manualUpdate(db) {
+            expectedRelationships.forEach(
+              ({ sourceLabel, targetLabel }) => {
+                const sourceId = IModelTestUtils.queryByUserLabel(db, sourceLabel);
+                const targetId = IModelTestUtils.queryByUserLabel(db, targetLabel);
+                assert(sourceId && targetId);
+                const rel = db.relationships.getInstance(
+                  ElementGroupsMembers.classFullName,
+                  { sourceId, targetId }
+                );
+                rel.delete();
+                db.elements.deleteElement(sourceId);
+                db.elements.deleteElement(targetId);
+              }
+            );
+          },
+        },
+      },
+      { branch: { sync: ["master"]}}, // first master->branch forward sync
+      {
+        assert({branch}) {
+          for (const rel of expectedRelationships) {
+            expect(branch.db.relationships.tryGetInstance(
+              ElementGroupsMembers.classFullName,
+              rel.idInBranch,
+            ), `had ${rel.sourceLabel}->${rel.targetLabel}`).to.be.undefined;
+            const sourceId = IModelTestUtils.queryByUserLabel(branch.db, rel.sourceLabel);
+            const targetId = IModelTestUtils.queryByUserLabel(branch.db, rel.targetLabel);
+            // Since we deleted both elements in the previous manualUpdate
+            assert(sourceId === "0" && targetId === "0", `SourceId is ${sourceId}, expected 0. TargetId is ${targetId}, expected 0.`);
```

Comment:
nit: more self documenting to do `=== Id64.invalid` instead of inline `0`

Reply:
changed to isinvalid